Description: remove few ASM
Author: Nicolas HOUDELOT <nicolas@demosdebs.org>
Applied-Upstream: no
Last-Update: 2025-02-10
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/scalemark.c
+++ b/scalemark.c
@@ -435,8 +435,7 @@
 	}
 }
 
-void _start() {
-	asm volatile("sub $8, %rsp\n");
+int main() {
 
 	generate_song();
 
@@ -485,4 +484,4 @@
 		SDL_GL_SwapWindow(mainwindow);
 	}
 	__builtin_unreachable();
-}
\ No newline at end of file
+}
